### ISSUE TITLE
NeoTree showing poor contrast

### ISSUE THREAD

**Issue by bini-x** [AlexvZyl/nordic.nvim#157]
Hi, I started using Nordic and I think it's a fantastic colorscheme, but I've had a problem with some NeoTree colors.
![Screenshot from 2024-09-23 11-42-31](https://github.com/user-attachments/assets/e54606e5-3163-4d70-8c02-64e025b570fc)
As you can see in this screenshot, text color of a git renamed file has some very low contrast (and the folders in which changes have been made). Also when I add a new file or dir through NeoTree, text also has very low contrast as shown in this image:
![image](https://github.com/user-attachments/assets/563ffad9-3562-4821-bf26-6957819f9e4f)

**Comment by bini-x** [comment]
i would say name of files to be white3 (#ECEFF4), background: black0
(#191D24), name of directories: gray5 (#60728A), renamed git files: gray5
or gray4 (#4C566A), text of when adding a file (the dialog): white3.
So basically: white3, black0 and gray5 (maybe gray4 if you see it
reasonably). If I wasn't clear enough feel free to contact me again and
thanks for considering the fix.

**Comment by AlexvZyl** [comment]
Agreed that it does not look good.  You can open a PR if you want?  It is pretty simple.

**Comment by bini-x** [comment]
Oh okay, sorry for misunderstanding.

**Comment by bini-x** [comment]
Yeah true, just didn't get that PR means Pull Request😅.
Now I'm a bit busy but later I'll try to figure it out. If I need any
help I'll let you know.
Thanks!

**Comment by bini-x** [comment]
Hi Alex!
Just wanted to let you know that I've never made a color scheme or plugin
for neovim, so I don't know if it's a good idea to do a PR.
Here's how I fixed it on my config:
[image: image.png]
Is there any way to add this piece of code on the github repo or not?

**Comment by bini-x** [comment]
Hey sorry I might be disturbing you but I think I found the way to add
these to your repo. Is this the piece of code I have to change (the part of
NeoTree):
[image: image.png]

**Comment by AlexvZyl** [comment]
You have to fork nordic, and then create a PR from your fork :)

**Comment by bini-x** [comment]
Thanks, I'll do that now

**Comment by bini-x** [comment]
[image: image.png]
if there's anything you don't like, feel free to let me know
